helper/testhelpers/testhelpers.go

```diff
@@ -820,16 +820,22 @@ func CreateEntityAndAlias(t testing.T, client *api.Client, mountAccessor, entity
 // CreateEntityAndAliasWithinMount clones an existing client and creates an entity/alias, within the specified mountPath
 // It returns the cloned client, entityID, and aliasID.
 func CreateEntityAndAliasWithinMount(t testing.T, client *api.Client, mountAccessor, mountPath, entityName, aliasName string) (*api.Client, string, string) {
+	return CreateCustomEntityAndAliasWithinMount(t, client, mountAccessor, mountPath, aliasName, map[string]interface{}{
+		"name": entityName,
+	})
+}
+
+// CreateCustomEntityAndAliasWithinMount clones an existing client and creates an entity/alias, using the custom entity, within the specified mountPath
+// It returns the cloned client, entityID, and aliasID.
+func CreateCustomEntityAndAliasWithinMount(t testing.T, client *api.Client, mountAccessor, mountPath, aliasName string, entity map[string]interface{}) (*api.Client, string, string) {
```

Comment:
Refactoring this helper allowed me to replace 2 large chunks of equivalent code in the Login MFA Okta tests with this helper.